stylelint.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Wow - really surprising a lint config didn't already exist!

Reply:
That's what I thought 🤷